test/integration/query-interface/schemas.test.ts

```diff
@@ -0,0 +1,47 @@
+import { expect } from 'chai';
+import { afterEach } from 'mocha';
+import { spy } from 'sinon';
+import type { CreateSchemaQueryOptions } from '@sequelize/core/_non-semver-use-at-your-own-risk_/dialects/abstract/query-generator';
+import { sequelize } from '../support';
+
+const queryInterface = sequelize.queryInterface;
+
+const testSchema = 'testSchema';
+
+if (sequelize.dialect.supports.schemas) {
+  describe('QueryInterface#{create}Schema', async () => {
+
+    // Clear the test schema if it's been added
+    afterEach(async () => {
+      const schemaNames = await queryInterface.showAllSchemas();
+      if (schemaNames.includes(testSchema)) {
+        await queryInterface.dropSchema(testSchema);
+      }
+    });
+
+    it('should create a schema', async () => {
```

Comment:
We don't follow the should convention because it's needlessly verbose

```suggestion
    it('creates a schema', async () => {
```